gnomad_qc/v4/create_release/validate_and_export_vcf.py

```diff
@@ -979,6 +983,9 @@ def main(args):  # noqa: D103
             ht = res.release_ht.ht()
             validated_ht = res.validated_ht.ht()
 
+            # v4 Genomes drops subsets from VCF
+            subsets = SUBSETS["exomes"] if data_type == "exomes" else []
```

Comment:
I don't know if you still need SUBSETS defined above for "exomes" and "genomes" since we are not including subsets in genomes VCF.

Reply:
We do because the code above needs the genome subsets for the validtion